tardis/io/atom_data/base.py

```diff
@@ -192,6 +237,10 @@ def from_hdf(cls, fname=None):
 
             try:
                 atom_data.version = store.root._v_attrs["database_version"]
+                if hasattr(atom_data.version, "decode"):
```

Comment:
Maybe we should get rid of this part now, as it's never encoded in the first place?

Reply:
I'd say yes, the `version` attribute is new and was never encoded. That's different for MD5 and UUID1 that existed in previous versions.